### ISSUE TITLE
Redundant setting of env property in app-start.js?

### ISSUE THREAD

**Issue by philbooth** [mozilla/fxa-content-server#6920]
The implementation of `ExperimentChoiceIndex::choose` takes care to [automatically propagate `subject.env` in calls to `choose`](https://github.com/mozilla/fxa-content-server/blob/9903562ae5b4f8c30d84d1e14b38597b2ad79a2b/app/scripts/lib/experiments/grouping-rules/index.js#L51-L54):

```js
const subjectCopy = Object.create(subject);
subjectCopy.env = subject.env || this._env;
subjectCopy.experimentGroupingRules = this;
return experiment.choose(subjectCopy);
```

And `app-start.js` takes care to [set it when creating `this._experimentGroupingRules`](https://github.com/mozilla/fxa-content-server/blob/9903562ae5b4f8c30d84d1e14b38597b2ad79a2b/app/scripts/lib/app-start.js#L93-L95):

```js
this._experimentGroupingRules = new ExperimentGroupingRules({
  env: this._config.env
});
```

But then it continues to set it explicitly later when it uses `this._experimentGroupingRules` too, which seems like it should be unnecessary:

* https://github.com/mozilla/fxa-content-server/blob/9903562ae5b4f8c30d84d1e14b38597b2ad79a2b/app/scripts/lib/app-start.js#L147-L151
* https://github.com/mozilla/fxa-content-server/blob/9903562ae5b4f8c30d84d1e14b38597b2ad79a2b/app/scripts/lib/app-start.js#L173-L176

**Comment by hritvi** [comment]
May I work on this issue @philbooth?

**Comment by philbooth** [comment]
@hritvi, yep, be my guest. Warning though that I only opened this based on eyeballing the code, so it may turn out that it's not redundant for some reason I didn't notice, or something.

Probably best to start off by removing it from the properties in the second two bullet points, then seeing what tests fail and whether we just need to fix up a couple of assertions or if it maybe breaks something more fundamental.
